ocs_ci/helpers/helpers.py

```diff
@@ -2995,6 +2995,25 @@ def get_mon_pdb():
     return disruptions_allowed, min_available_mon, max_unavailable_mon
 
 
+def verify_pdb_mon(disruptions_allowed, max_unavailable_mon):
+    """
+    Verify PDB mon
```

Comment:
Please elaborate the description explaining more about this function

Reply:
done